### PR TITLE
Rover : Included another crash check in crash_check.cpp

### PR DESCRIPTION
**Pull request by sarthak268**
With reference to #7899

### CODE REVIEW

File: APMrover2/crash_check.cpp [hunk 1]
```diff
@@ -21,15 +22,32 @@ void Rover::crash_check()
...
-    crash_counter = 0;
-    return;
-  }
+      crash_counter = 0;
+      bool crash_checker = true;
+
+
+  float lean_angle = degrees(safe_sqrt(ahrs.pitch*ahrs.pitch + ahrs.roll*ahrs.roll));
+  if (lean_angle > CRASH_ANGLE) {
+    crash_counter ++;
```
Comment (muramura) [inline APMrover2/crash_check.cpp]: The indentation in the coding style is 4 white Spaces. There are other lines as well.

File: APMrover2/crash_check.cpp [hunk 2]
```diff
@@ -43,4 +61,4 @@ void Rover::crash_check()
       disarm_motors();
     }
   }
-}
+}
```
Comment (amilcarlucas) [inline APMrover2/crash_check.cpp]: This change should not be commited

File: APMrover2/crash_check.cpp [hunk 3]
```diff
@@ -4,6 +4,7 @@
 static const uint16_t CRASH_CHECK_TRIGGER_SEC = 2;   // 2 seconds blocked indicates a crash
 static const float CRASH_CHECK_THROTTLE_MIN = 5.0f;  // vehicle must have a throttle greater that 5% to be considered crashed
 static const float CRASH_CHECK_VEL_MIN = 0.08f;      // vehicle must have a velocity under 0.08 m/s or rad/s to be considered crashed
+static const float CRASH_ANGLE = 10;
```
Comment (amilcarlucas) [inline APMrover2/crash_check.cpp]: Please add a comment. What angle is it ? relative to what ?

**Comment by rmackay9** [comment]
The new crash check has gone in as part of [this other PR](https://github.com/ArduPilot/ardupilot/pull/8701) so closing this PR.  Thanks anyway!

**Pull request closed without merging**